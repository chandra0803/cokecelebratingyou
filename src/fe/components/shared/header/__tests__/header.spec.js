import React from 'react';
import Header from '../header.js';
import renderer from 'react-test-renderer';

test( 'Header renders differently based on the "altText" prop', () => {
    const component = renderer.create(
        <div>
            <Header altText="one alt text" />
            <Header altText="another alt text" />
        </div>
    );

    const tree = component.toJSON();
    expect( tree ).toMatchSnapshot();
} );
