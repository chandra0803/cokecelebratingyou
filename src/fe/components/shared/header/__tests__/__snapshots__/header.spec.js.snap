// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders differently based on the "altText" prop 1`] = `
<div>
  <header
    className="site-header"
    id="header"
  >
    <h1
      className="logo"
    >
      <a
        href="homePage.do"
      >
        <img
          alt="one alt text"
          src="assets/skins/undefined/img/logo-primary.png"
        />
      </a>
    </h1>
  </header>
  <header
    className="site-header"
    id="header"
  >
    <h1
      className="logo"
    >
      <a
        href="homePage.do"
      >
        <img
          alt="another alt text"
          src="assets/skins/undefined/img/logo-primary.png"
        />
      </a>
    </h1>
  </header>
</div>
`;
